Makefile

```diff
@@ -40,7 +40,7 @@ define run_test
 endef
 
 define make_docker
-	$(shell [ -z "$(SERVICE)" ] && SERVICE=$(subst docker_,,$(1)))
+	$(eval SERVICE=$(shell [ -z "$(SERVICE)" ] && echo $(subst docker_,,$(1)) || echo $(SERVICE)))
```

Comment:
what is the purpose of these changes?

Reply:
This enables the running locally the goal 
`make dockers` and all services will build successfully, before it was erroring out, since it in the pipeline it receives the SERVICE env through the command line